docker-files/local/django/Dockerfile

```diff
@@ -39,4 +35,3 @@ RUN chmod +x /start-flower
 
 WORKDIR /app
 
-ENTRYPOINT ["/entrypoint"]
```

Comment:
why remove this? we don't to make connection the db anymore?
where is the alternative cmd for this?

Reply:
This is not making the connection to the database , it was just the entrypoint for the application.